patroni/api.py

```diff
@@ -140,6 +152,22 @@ def do_GET(self, write_status_code_only=False):
             elif path in ('/async', '/asynchronous') and not is_synchronous:
                 status_code = replica_status_code
 
+        # check for user defined tags in query params
+        if status_code == 200:
+            qs_tag_prefix = "tag_"
+            qs_tags = {tag_name[len(qs_tag_prefix):]: tag_values[0]
+                       for tag_name, tag_values in self.path_query.items()
+                       if tag_name.startswith(qs_tag_prefix)}
+            for qs_key, qs_value in qs_tags.items():
```

Comment:
Is there a need to do a list comprehension if later we still have a for loop over the just built dict?

Reply:
oops:D